src/classes/Trades.ts

```diff
@@ -293,9 +293,9 @@ export default class Trades {
         action: 'accept' | 'decline' | 'skip',
         reason: string,
         meta: Meta,
-        offer: TradeOfferManager.TradeOffer
+        offer: TradeOffer
     ): Promise<void> {
-        this.bot.handler.onOfferAction(offer, action, reason, meta);
+        void this.bot.handler.onOfferAction(offer, action, reason, meta);
```

Comment:
this is effectively saying, start the onOfferAction promise and continue immediately. do we want to wait for this promise to finish?

Reply:
hurm not sure.